ui/admin-portal/src/app/components/candidates/view/shareable-docs/shareable-docs.component.html

```diff
@@ -8,38 +8,40 @@
 </div>
 <div *ngIf="candidateSource" class="text-muted mb-3">If left blank, the candidate's default selected shareable documents will be used (if present).</div>
 
-<form [formGroup]="form">
-  <div  class="flex-wrap justify-content-around">
-    <div class="d-flex flex-grow-1 min-width-300 align-items-center justify-content-start mb-2">
+<div *ngIf="this.cvs && this.other">
```

Comment:
Do you understand why neither this.cvs and this.other can be true - I thought it always should be one or the other

Reply:
They are two seperate filtered arrays from all the attachments, filtered by CV true/false. So both arrays should exist, but they could be empty. Perhaps this isn't right, but that was my thinking. Now we are postponing this bug to next sprint I can have another look